src/Paket.Core/embedded/Paket.Restore.targets

```diff
@@ -102,7 +102,11 @@
       <PaketRestoreRequired Condition=" '$(PaketRestoreLockFileHash)' == '' ">true</PaketRestoreRequired>
     </PropertyGroup>
 
-    <PropertyGroup Condition="'$(PaketPropsVersion)' != '5.174.2' ">
+	<!-- 
+		This value should match the version in the props generated by paket 
+		If they differ, this means we need to do a restore in order to ensure correct dependencies
+	-->
+    <PropertyGroup Condition="'$(PaketPropsVersion)' != '5.185.3' ">
```

Comment:
I don't think this is correct. This is the props file format version  - and it did not change recently.

Reply:
maybe we need to check if it's not older than the version in the targets file. dunno

Reply:
this commit is when the props version was changed. https://github.com/fsprojects/Paket/commit/828f9e4553d4d242be603e72a08cdec0c5fb454f
This file was not updated at that time.